packages/contracts-bedrock/src/safe/DeputyPauseModule.sol

```diff
@@ -13,7 +13,6 @@ import { ECDSA } from "@openzeppelin/contracts-v5/utils/cryptography/ECDSA.sol";
 
 // Interfaces
 import { ISemver } from "interfaces/universal/ISemver.sol";
-import { IDeputyGuardianModule } from "interfaces/safe/IDeputyGuardianModule.sol";
 import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
 
 /// @title DeputyPauseModule
```

Comment:
Several comments in this file need to be updated, this is now intended to be installed directly into the Guardian safe